packages/pds/tests/moderation.test.ts

```diff
@@ -568,8 +568,8 @@ describe('moderation', () => {
 
     it('only allows blob to have one current action.', async () => {
       const img = sc.posts[sc.dids.carol][0].images[0]
-      const postA = await sc.post(sc.dids.alice, 'image A', undefined, [img])
-      const postB = await sc.post(sc.dids.alice, 'image B', undefined, [img])
+      const postA = await sc.post(sc.dids.carol, 'image A', undefined, [img])
+      const postB = await sc.post(sc.dids.carol, 'image B', undefined, [img])
```

Comment:
Just curious what called for this change.

Reply:
these blobs were uploaded by carol so we needed to use them in her posts instead.

since we're associating blobs with users now!